### PR TITLE
Add pyanalyze and typeshed to black-primer

### PR DESCRIPTION
**Pull request by JelleZijlstra**
pyanalyze is one of my projects and it uses `--experimental-string-processing`.

typeshed has a lot of stub files.

### CODE REVIEW

File: src/black_primer/primer.json [hunk 1]
```diff
@@ -75,6 +75,13 @@
       "long_checkout": false,
       "py_versions": ["all"]
     },
+    "pyanalyze": {
+      "cli_arguments": [],
```
Comment (cooperlees) [inline src/black_primer/primer.json:79]: I think we need to add `--experimental-string-processing` here
```suggestion
      "cli_arguments": ["--experimental-string-processing"],
```

File: src/black_primer/primer.json [hunk 2]
```diff
@@ -110,6 +117,13 @@
       "long_checkout": false,
       "py_versions": ["all"]
     },
+    "typeshed": {
```
Comment (cooperlees) [inline src/black_primer/primer.json:120]: Nice! Didn't know type shed had python we would format!

**Review by cooperlees — CHANGES_REQUESTED** [review]

File: src/black_primer/primer.json (hunk 1 above)
Comment (JelleZijlstra) [reply src/black_primer/primer.json:79]: It's in the pyproject.toml, does that not get picked up?

File: src/black_primer/primer.json (hunk 2 above)
Comment (JelleZijlstra) [reply src/black_primer/primer.json:120]: We can format stubs too! That makes typeshed a good way to test our stub formatting.

**Review by cooperlees — APPROVED** [review]

**Pull request merged**